site/content/learn/headless/request-interception.md

```diff
@@ -45,13 +45,13 @@ In the following snippet we are going to abort all requests for images on our te
 
 {{< tabs "2" >}}
 {{< tab "Playwright" >}}
-```js {9-13}
+```js {hl_lines=["11-13"]}
```

Comment:
@pilimartinez could we include the outer block, too? for both Puppeteer and Playwright